client/src/main/java/io/undertow/client/http/Http2ClientCompletableFutureNoRequest.java

```diff
@@ -24,6 +24,9 @@ public void completed(final ClientExchange result) {
 
                     @Override
                     protected void stringDone(String string) {
+//                        if (logger.isDebugEnabled()) {
+//                            logger.debug("Service call response = {}", string);
+//                        }
```

Comment:
Is there a reason we need to change the Undertow copied classes? Normally they would be under a different log level anyway..

Reply:
Once adding this logger...The travis test won't pass. Need further research to find the reason.